packages/nocodb/src/models/LongTextColumn.ts

```
@@ -0,0 +1,113 @@
+    if (!column) {
+      NcError.fieldNotFound(insertObj.fk_column_id);
+    }
+
+    await ncMeta.metaInsert2(
+      context.workspace_id,
+      context.base_id,
+      MetaTable.COL_LONG_TEXT,
+      insertObj,
+    );
+
+    return this.read(context, longTextColumn.fk_column_id, ncMeta);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)